Quantori.AllureTestOpsClient/Model/CustomFieldItem.cs

```diff
@@ -2,7 +2,7 @@
 {
     public class CustomFieldItem
```

Comment:
Could you please add a xml summary to this class?

Reply:
This is just a DTO